src/apollos/models/users/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export default axios.create({
+  baseURL: process.env.ROCK_URL,
+  headers: {
+    [process.env.ROCK_TOKEN_NAME]: process.env.ROCK_TOKEN, // Not too sure about this
```

Comment:
I didn't see `ROCK_TOKEN_NAME` in the `env.example` file but it is somehow registered in holtzman. Does it work this way? Or is the token name static?

Reply:
@Lepozepo It's static. It's always just "Authorization-Token". You can see that we hard code it in Heighliner [here](https://github.com/NewSpring/Heighliner/blob/master/src/rock/mssql.js#L108). 

Something I noticed is that we specify the token name in the Holtzman `settings.json` file in the Meteor folder (along with the URL and the token), but in Heighliner, we only specify the URL and the token. Perhaps it would benefit us, in the long run, to add the token name to the `.env` file in Heighliner and reference it that way instead of hardcoding it?

Reply:
I think if it's something admins can modify on Rock then it should be configurable but otherwise having it be hard-coded could make it easier for people to pick up and set up. I'll add it to the env example in the meantime though